### PR TITLE
[PR-8776] Document liquid date/time filters

### PR DESCRIPTION
**Pull request by Doimhneacht**
## Demo

http://void-docs.talkable.com/campaigns/editor/filters.html?highlight=filters#date-diff

## Related Stories

[](https://talkable.atlassian.net/browse/PR-8776)

### CODE REVIEW

File: source/campaigns/editor/filters.rst [hunk 1]
```diff
@@ -182,6 +182,81 @@ A/B testing style:
...
+
+.. code-block:: text
+
+   {{ valid_until | date_diff }}
+   {{ valid_until | date_diff: 'Apr 4, 2019' }}
+   {{ '2019-09-01 08:30:00' | date_diff: current_time }}
+
+|hr|
+
+date\_more\_than
```
Comment (zhuravel) [inline source/campaigns/editor/filters.rst]: I think we should change more_than to greater_than – that is proper English and that is how it is named in Rails validators, for example.

I propose to rename:
`date_more_than` → `date_greater_than` (or `date_later_than`)
`date_more_than_or_equal` → `date_greater_than_or_equal` (or `date_later_than_or_equal`)

"More than" sounds unnatural to me when comparing dates.
@vitalikdanchenko @iurevych @Doimhneacht What do you think?

Comment (Doimhneacht) [reply source/campaigns/editor/filters.rst]: I think `date_greater_or_equal` sounds better than `date_later_or_equal`

Comment (iurevych) [reply source/campaigns/editor/filters.rst]: I agree that `greater` sounds more native than `more` when applied to such context.

File: source/campaigns/editor/filters.rst [hunk 2]
```diff
@@ -182,6 +182,81 @@ A/B testing style:
...
+.. code-block:: text
+
+   {{ valid_until | date_diff }}
+   {{ valid_until | date_diff: 'Apr 4, 2019' }}
+   {{ '2019-09-01 08:30:00' | date_diff: current_time }}
+
+|hr|
+
+date\_greater\_than
+................
```
Comment (zhuravel) [inline source/campaigns/editor/filters.rst]:
```suggestion
...................
```

File: source/campaigns/editor/filters.rst [hunk 3]
```diff
@@ -182,6 +182,81 @@ A/B testing style:
...
+
+.. code-block:: liquid
+
+   {{ current_time | date_greater_than: valid_until }}
+   {{ valid_until | date_greater_than: '2019-09-01 08:30:00' }}
+
+|hr|
+
+date\_greater\_than\_or\_equal
+...........................
```
Comment (zhuravel) [inline source/campaigns/editor/filters.rst]:
```suggestion
..............................
```

**Pull request merged**